src/app/modules/main/profile_section/profile/models/profile_preferences_collectible_item.nim

```diff
@@ -1,19 +1,40 @@
-import json, strutils, stint, json_serialization, tables
+import json, strutils, strformat, stint, json_serialization, tables
 
 import profile_preferences_base_item
 
 import app_service/service/profile/dto/profile_showcase_preferences
+import app/modules/shared_models/collectibles_entry
 
 include app_service/common/json_utils
 include app_service/common/utils
 
 type
   ProfileShowcaseCollectibleItem* = ref object of ProfileShowcaseBaseItem
-    uid*: string
+    chainId*: int
+    tokenId*: string
+    contractAddress*: string
+    communityId*: string
     name*: string
     collectionName*: string
     imageUrl*: string
     backgroundColor*: string
+    loading*: bool
+
+proc initProfileShowcaseCollectibleItem*(collectible: CollectiblesEntry, visibility: ProfileShowcaseVisibility, order: int): ProfileShowcaseCollectibleItem =
+  result = ProfileShowcaseCollectibleItem()
+  result.contractAddress = collectible.getContractAddress()
+  result.chainId = collectible.getChainID()
+  result.tokenId = collectible.getTokenIDAsString()
+  result.communityId = collectible.getCommunityId()
+  result.name = collectible.getName()
+  result.collectionName = collectible.getCollectionName()
+  result.imageUrl = collectible.getImageURL()
+  result.backgroundColor = collectible.getBackgroundColor()
+  result.showcaseVisibility = visibility
+  result.order = order
+  result.showcaseVisibility = visibility
```

Comment:
@MishkaRogachev this and the following property are repeated.

Reply:
fixed